docs/woocommerce.com/analytics-customers-report.md

```diff
@@ -12,7 +12,7 @@ This option loads the report for the customer you have selected.
 
 ![Customers Report Single Customer View](images/analytics-customers-report-single-customer.png)
 
-### Advanced Filters
+### Advanced filters
```

Comment:
Should the documentation headings stay capitalized?

Reply:
@adrianduffell I'd say since we're sentence casing all the things in product, it's okay to sentence case them in our docs too.

Reply:
I'm not sure. I feel like a documentation heading should remain title case since it's the more common style for this context.

This probably needs a second opinion 🙂 

cc @woocommerce/isotope 

If we change this instance to sentence case, for consistency I think all of the other documentation headings need to be changed too. There are still instances of title case in this file and elsewhere. For example:

<img width="1065" alt="Screen Shot 2021-03-30 at 10 09 11 am" src="https://user-images.githubusercontent.com/9312929/112923010-0dc6d100-9140-11eb-80e9-28fa070966e2.png">

<img width="1030" alt="Screen Shot 2021-03-30 at 10 09 15 am" src="https://user-images.githubusercontent.com/9312929/112923013-10292b00-9140-11eb-9649-4db518c28e30.png">

<img width="1062" alt="Screen Shot 2021-03-30 at 10 09 25 am" src="https://user-images.githubusercontent.com/9312929/112923022-13bcb200-9140-11eb-8c5f-257ad7b438ea.png">

<img width="755" alt="Screen Shot 2021-03-30 at 10 09 29 am" src="https://user-images.githubusercontent.com/9312929/112923032-17e8cf80-9140-11eb-8a84-266655bf674f.png">

Reply:
I also think the documentation heading should remain title case as all other docs in our repository use title case.